benchopt/runner.py

```diff
@@ -196,7 +196,8 @@ def run_one_solver(benchmark, dataset, objective, solver, n_repetitions,
             solver_name=str(solver),
             data_name=str(dataset),
             idx_rep=rep,
-            stopping_strategy=stopping_strategy.capitalize()
+            stopping_strategy=stopping_strategy.capitalize(),
+            solver_description=str(solver.__doc__) or ""
```

Comment:
100% cosmetic: if you add a comma at the end of the line, you'll make future diffs easier (notice here the diff is two lines because you had to add a comma to the previously last line, whereas essentially it should be a single one)

Reply:
I believe str(solver.__doc__) evaluates to `"None"` when there is no docstring so it's True and we never enter the part after the `or` when the solver has no docstring

You can achieve whet you wanted with `str(solver.__doc__) if solver.__doc__ is not None else ""`

Reply:
yes indeed, thanks!